js/main.js

```diff
@@ -0,0 +1,314 @@
+const buttons = document.getElementsByClassName("button");
+const digitButtons = document.getElementsByClassName("digitButton");
+const screen = document.getElementById("screen");
+const unblockedKeys = [8, 13, 16, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 114, 187];
+
+let calcType = null;
+let result = null;
+let factor2= null;
```

Comment:
I'm not sure, but if I understand correct idea, then better to name this variable like `previousIterationResult` or `previousIterationMathResult`.

Reply:
avoid names like `result3` or `person25`, not clear why in some situation we have the third result and why the number of some person is 25.

Reply:
in some cases, math operations in the code require only one factor. in these cases, i use the "result" variable. in other cases, two factors are needed. therefore i use "result" as the first factor and "factor2" as the second factor, e.g.:
factor1 + factor2 = result
for "factor1" i use "result" 
for "+" i use "mathOperation"("calcType")
for "factor2" i use "factor2"
the value of "result" i assign to "result" too

However, as you wish. I'll call it "additionalFactor"